bin/retrieve_eload_from_lts.py

```diff
@@ -0,0 +1,149 @@
+import fileinput
+import logging
+import os
+from argparse import ArgumentParser
+
+import yaml
+from ebi_eva_common_pyutils import command_utils
+
+logging.basicConfig(format='%(message)s', level=logging.DEBUG)
```

Comment:
Because this is in `eva-submission` I would prefer to use the same as the other script in bin like importing logging_config and load_config

```python
from ebi_eva_common_pyutils.logger import logging_config as log_cfg
from eva_submission.submission_config import load_config
```